DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
Eventually means not yet done...





On Jul 7, 2015, at 2:02 PM, Ren-Jie Wang <notifications@github.com<mailto:notifications@github.com>> wrote:


In DQM/L1TMonitor/python/L1TCSCTF_cfi.py<https://github.com/cms-sw/cmssw/pull/10011#discussion_r34031964>:

> @@ -17,5 +17,6 @@
>  #
>  # Make changes for running in Run 2
>  #
> -from Configuration.StandardSequences.Eras import eras
> -eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
> +# This is not working.
> +#from Configuration.StandardSequences.Eras import eras
> +#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )


I did read your comment: "which will eventually be replaced by the eras.run2", but is this true ? Why do we still see the CSCTF DQM plots with run 1 configuration if it will eventually be replaced by the eras.run2 ?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/10011/files#r34031964>.